website/docs/cloud-docs/api-docs/agents.mdx

```diff
@@ -384,10 +403,13 @@ This POST endpoint requires a JSON object with the following properties as a req
 
 Properties without a default value are required.
 
-| Key path               | Type   | Default | Description                                                                                                                                                   |
-| ---------------------- | ------ | ------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------- |
-| `data.type`            | string |         | Must be `"agent-pools"`.                                                                                                                                      |
-| `data.attributes.name` | string |         | The name of the agent-pool, which can only include letters, numbers, `-`, and `_`. This will be used as an identifier and must be unique in the organization. |
+| Key path                                          | Type   | Default | Description                                                                                                                                                                  |
+|---------------------------------------------------|--------|---------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `data.type`                                       | string |         | Must be `"agent-pools"`.                                                                                                                                                     |
+| `data.attributes.name`                            | string |         | The name of the agent-pool, which can only include letters, numbers, `-`, and `_`. This will be used as an identifier and must be unique in the organization.                |
+| `data.attributes.organization-scoped`             | bool   | true    | The scope of the agent-pool. If true then the agent pool is allowed to be used by all workspaces in the organization. Available in Terraform Enterprise v202207-1 and later. |
+| `data.relationships.allowed-workspaces.data.type` | string |         | Must be `"workspaces"`. Available in Terraform Enterprise v202207-1 and later.                                                                                               |
+| `data.relationships.allowed-workspaces.data.id`   | string |         | The ID of the workspace that is allowed to use the agent pool. Available in Terraform Enterprise v202207-1 and later.                                                        |
```

Comment:
When we say "allowed", are we referencing scoping? 

I wonder if we should add "Refer to [Scoping Agent Pools to Specific Workspaces](/docs/cloud-docs/agents#scoping-agent-pools-to-specific-workspaces) for details.

Also - can you scope it to only ONE workspace, or more than one?

Reply:
Yes, it refers to scoping and we can scope of an agent pool to multiple workspaces